pkg/util/docker/containers.go

```diff
@@ -218,6 +218,10 @@ func parseContainerNetworkAddresses(ports []types.Port, netSettings *types.Summa
 		}
 	}
 	for _, network := range netSettings.Networks {
+		if network.IPAddress == "" {
+			log.Debugf("No IP found for container %s in network %s", container, network.NetworkID)
```

Comment:
Due to the potential verbosity might even be `Tracef` WDYT?

Reply:
I tried to be consistent with log levels I saw in https://github.com/DataDog/datadog-agent/blob/33db6c888082da73e04a7d344b9c78ee3a72371d/pkg/util/docker/network.go. I think it will most of the time appear once per container for k8s setups every 10s, but it may appear more in other setups. Do you think the latter qualifies for downgrading to trace?